obs-check-devel-openqa-leap-extra-dependencies

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
+
+osc="osc --apiurl https://api.opensuse.org"
+
+for project in $($osc search --project -s 'devel:openQA:Leap:' | grep -o '^devel:openQA:Leap:.*'); do
+    for package in $($osc list "$project"); do
+        $osc api "/comments/package/$project/$package" | \
+            yq -pxml -oy '.comments.comment |= ([] + .) | .comments.comment[].+content' | \
+            grep -q "Reason for linking:" || \
+            { echo "No reason for $project/$package" >&2; problem=1; }
```

Comment:
Where is this going to be used? And do you have some example output to show that it works?

Reply:
> Where is this going to be used?

Something like a script on schedule that informs us about such inefficient use of packages